tests/eas/acceptance.py

```diff
@@ -44,6 +44,10 @@ def local_setup(env):
         env.target.execute(
             "echo ENERGY_AWARE > /sys/kernel/debug/sched_features")
 
+    if SET_IS_BIG_LITTLE:
+        is_big_little_sysctl = "/proc/sys/kernel/sched_is_big_little"
+        if (env.target.file_exists(is_big_little_sysctl)):
```

Comment:
The best devlib's API to write sysfs attributes is write_value():
   https://github.com/ARM-software/devlib/blob/master/devlib/target.py#L322
which provides also a check for the value being properly set.

Moreover, since it's a kind-of "brute force" setup, maybe we can avoid the two ADB commands (check and write) by just doing:
```python
if SET_IS_BIG_LITTLE:
    try:
        te.target.write_value(is_big_little_sysctl, 1)
    except: pass
```